experimental/personnames/src/api.rs

```diff
@@ -31,13 +34,15 @@ pub trait PersonName {
 pub enum PersonNamesFormatterError {
     ParseError(String),
     InvalidPersonName,
+    InvalidLocale,
+    InvalidCldrData(String),
```

Comment:
this is not a good name, because the data is not necessarily CLDR data by the time you read it here. `InvalidData` should be good enough.